src/fragment-components/a-code-snippet.tsx

```diff
@@ -104,11 +118,13 @@ export const componentInfo: ComponentInfo = {
 	codeExport: {
 		angular: {
 			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Type = "${json.variant}"
+			@Input() ${nameStringToVariableString(json.codeContext?.name)}Theme = "${json.light ? 'light' : 'dark'}"
```

Comment:
This isn't taking affect in Angular. Please update the angular export package.json and use the latest version of CCA. This will add `light` theme input